### ISSUE TITLE
Requires statements in all examples doesnt reflect real world

### ISSUE THREAD

**Issue by ericnewton76** [webpack/webpack#4767]
**Do you want to request a *feature* or report a *bug*?**
Feature/Change

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

In your examples, the first set of requires is something like this:
```
var XXXPlugin = require("../../lib/optimize/XXXPlugin");
```
And these require specs is not actual code that anybody would use in THEIR project... as newbie to node i didnt get this until a more senior node guy said, yeah thats wierd, heres probably what they meant.

**What is the expected behavior?**

Would show a real requires statement that can be used:
```
var AggressiveMergingPlugin = require("webpack").optimize.AggressiveMergingPlugin;
```

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**

Basically, I recommend splitting examples into a separate repo "webpack/webpack-examples" and providing a package.json that you would npm install webpack & npm build [example] or something much more similar to how it would install into actual projects.  This way the examples will be better at illustrating the correct ways to integrate.

Thx

**Comment by sokra** [comment]
The default dev process for webpack is the `npm link webpack` into itself so we could just change the examples and it would work fine.

Do you want to do a PR? Examples are integration tested so you can't do it wrong...
